test/Microsoft.Sbom.Parsers.Spdx22SbomParser.Tests/Parser/SbomRelationshipParserTests.cs

```diff
@@ -0,0 +1,136 @@
+﻿using Microsoft.Sbom.Exceptions;
+using Microsoft.Sbom.Parser.Strings;
+using Microsoft.Sbom.Parsers.Spdx22SbomParser;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.IO;
+using System.Text;
+
+namespace Microsoft.Sbom.Parser;
+
+[TestClass]
+public class SbomRelationshipParserTests
+{
+    [TestMethod]
+    public void ParseSbomRelationshipsTest()
+    {
+        byte[] bytes = Encoding.UTF8.GetBytes(RelationshipStrings.GoodJsonWith2RelationshipsString);
+        using var stream = new MemoryStream(bytes);
+
+        TestParser parser = new ();
+        var count = 0;
+
+        foreach (var relationship in parser.GetRelationships(stream))
+        {
+            count++;
```

Comment:
What exactly is this doing?  I see the assert below, but maybe the more meanginful assertion is to do something like the following.  It's not a huge deal but it feels like you're testing something outside of the actual SUT.

`
var relationships = parser.GetRelationships(stream);
...
Assert.AreEqual(2, relationships.Count());
`

Reply:
I will use that library for all the tests in the final PR